app/models/user.rb

```diff
@@ -241,6 +241,27 @@ def state_file_admin?
     role_type == AdminRole::TYPE && role.state_file?
   end
 
+  def role_display_name
+    case role_type
+    when "SiteCoordinatorRole"
+      "Site Coordinator"
+    when "OrganizationLeadRole"
+      "Organization Lead"
+    when "TeamMemberRole"
+      "Team Member"
+    when "CoalitionLeadRole"
+      "Coalition Lead"
+    when "AdminRole"
+      "Admin"
+    when "ClientSuccessRole"
+      "Client Success"
+    when "GreeterRole"
+      "Greeter"
+    else
+      ""
+    end
+  end
+
```

Comment:
Did this get pulled in from Kevin's PR? Should this be internationalized?

Reply:
No it didn't, did he do something similar? I can rebase and about internationalization, good point it should

Reply:
I'm doing a similar look up in [my PR](https://github.com/codeforamerica/vita-min/pull/4664/files#diff-9802ca3c9c4cf89904fd44bc114e35ebdf2c5dd3d5b645491e2b253e1afef29bR256-R287), but for a different purpose, carry on 🙂

NIT, though: In most other places we use eg `SiteCoordinatorRole::TYPE`.